packages/ckeditor5-find-and-replace/src/findandreplaceui.js

```diff
@@ -88,7 +88,7 @@ export default class FindAndReplaceUI extends Plugin {
 			this.formView = formView;
 
 			editor.keystrokes.set( 'Ctrl+F', ( data, cancelEvent ) => {
-				dropdown.buttonView.actionView.fire( 'execute' );
```

Comment:
This has been fixed couple of days ago #10189 I'll merge the latest master branch.